edx_adapt/api/resources/course_resources.py

```diff
@@ -180,4 +180,37 @@ def post(self, course_id):
             self.repo.post_experiment(course_id, args['experiment_name'], args['start_time'], args['end_time'])
         except DataException as e:
             print("--------------------\tDATA EXCEPTION: " + str(e))
-            abort(500, message=str(e))
\ No newline at end of file
+            abort(500, message=str(e))
+
+prob_parser = reqparse.RequestParser()
+prob_parser.add_argument('prob_list', type=list, location='json', help="Please supply list with default probabilities")
+
+
+class Probabilities(Resource):
+    def __init__(self, **kwargs):
+        self.repo = kwargs['data']
+
+    def get(self, course_id):
+        prob_list = []
+        try:
+            prob_list = self.repo.get_probabilities(course_id)
+        except DataException as e:
+            abort(404, message=str(e))
+
+        return {'probabilities': prob_list}, 200
+
+    def post(self, course_id):
+        args = prob_parser.parse_args()
+        try:
+            self.repo.post_probabilities(course_id, args['prob_list'], new=True)
+        except DataException as e:
+            abort(500, message=str(e))
+        return {'success': True}, 200
```

Comment:
Semantically it's better to return 201 code here, rather than 200.

> 201 Created
> The request has been fulfilled and a new resource has been created.